tests/gateway/test_manage.py

```diff
@@ -17,6 +18,18 @@ def test_run_user_interruption(self, mock_aprs_client):
         self.assertEqual(instance.run.call_count, 1)
         instance.disconnect.assert_called_once_with()
 
+    # try to import stored OGN logfile
+    @mock.patch('ogn.gateway.manage.import_logfile')
+    def test_run_import_logfile(self, mock_import_logfile):
+        # instance = mock_import_logfile.return_value
+
+        # import_logfile(ogn_logfile="tests/OGN_log.txt_2016-09-21", reference_date="2016-09-21")
+
+        # instance.connect.assert_called_once_with()
+        # self.assertEqual(import_logfile.call_count, 1)
+        # instance.disconnect.assert_called_once_with()
+        pass
```

Comment:
I deactivated this test, because I couldn't make it work. I recommend disregarding the changes I made to  tests/gateway/test_manage.py